Sources/SmileID/Classes/SmileID.swift

```diff
@@ -64,6 +65,37 @@ public class SmileID {
         SmileID.useSandbox = useSandbox
     }
 
+    /// Set offline mode on the sdkJobStatusResponse
+    /// - Parameter allowOfflineMode: A boolean to enable offline mode or not
+    public class func setAllowOfflineMode(allowOfflineMode: Bool) {
+        SmileID.allowOfflineMode = allowOfflineMode
+    }
+
+    /// Returns a list of all unsubmitted jobIds
+    public class func getUnsubmittedJobs() -> [String] {
+        LocalStorage.getUnsubmittedJobs()
+    }
+
+    /// Returns a list of all submitted jobIds
+    public class func getSubmittedJobs() -> [String] {
+        LocalStorage.getSubmittedJobs()
+    }
+
+    /// nukes all offline stored files
+    public class func cleanup() throws {
+        try LocalStorage.deleteAll()
+    }
+
+    /// deletes the passed job id, whether in unsubmitted or submitted state
+    public class func cleanup(jobId: String) throws {
+        try cleanup(jobIds: [jobId])
```

Comment:
It is unfortunate that Swift does not support splatting -- that would have allowed us to have 2 methods here instead of 3.

I noticed we could easily provide more functionality here by making this a variadic function.

```suggestion
    public class func cleanup(jobIds: String...) throws {
        try cleanup(jobIds: jobIds)
```

This would allow any of the following to be valid:
```swift
try cleanup(jobIds: "job_1")
try cleanup(jobIds: "job_1", "job_2")
try cleanup(jobIds: ["job_1", "job_2"])
```

Reply:
We could also do something like this:
```swift
    /// deletes the job ids list, whether in unsubmitted or submitted state
    public class func cleanup(jobIds: [String]? = nil) throws {
        if let jobIds {
            try LocalStorage.delete(at: jobIds)
        } else {
            try LocalStorage.deleteAll()
        }
    }

    public class func cleanup(jobIds: String...) throws {
        try cleanup(jobIds: jobIds)
    }
```

Reply:
cleaned up